Library/Formula/mesos.rb

```diff
@@ -12,19 +12,22 @@ class Mesos < Formula
   depends_on :java => "1.7+"
   depends_on :macos => :mountain_lion
   depends_on "maven" => :build
-  # Use our Zookeeper for Yosemite and not the one shipped with Mesos
-  # Remove with next release.
-  # See https://github.com/Homebrew/homebrew/issues/32965
-  depends_on "zookeeper" if MacOS.version == :yosemite
+  depends_on :apr => :build
+  depends_on "subversion"
+  
+
+  needs :cxx11
 
   def install
     args = ["--prefix=#{prefix}",
             "--disable-debug",
             "--disable-dependency-tracking",
             "--disable-silent-rules",
+            "--with-svn=/usr/local/Cellar/subversion/1.8.11",
```

Comment:
Thanks.

Theoretically the configure script should be checking along the ` $PATH ` natively, if it's behaving as you'd expect it to. In which case, unless the user has no CLT installed, it *should* find the apr it desires. If that isn't working though, we need to adopt something very similar to [this fun block of code](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/subversion.rb#L137-L144).